admin_app_2/src/components/ContentCard.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import {
+  Card,
+  Typography,
+  Modal,
+  Toolbar,
+  Button,
+  Box,
+  Chip,
+} from "@mui/material";
+import { Edit, Translate } from "@mui/icons-material";
+import { LANGUAGE_OPTIONS, appColors, appStyles, sizes } from "@/utils";
+import { Layout } from "./Layout";
+import ContentViewModal from "@/components/ContentModal";
+import Link from "next/link";
+
+const ContentCard = ({
+  title,
+  contentID,
+}: {
+  title: string;
+  contentID: string;
+}) => {
+  const [open, setOpen] = React.useState<boolean>(false);
+  console.log("open", title, contentID);
+  return (
+    <>
+      <Card
+        sx={[
+          {
+            m: sizes.smallGap,
+            p: sizes.baseGap,
+          },
+          appStyles.shadow,
+        ]}
+      >
+        <Layout.FlexBox flexDirection={"row"} justifyContent={"space-between"}>
+          <Typography variant="body2">#{contentID}</Typography>
+          <Chip label="Pollution" variant="outlined" size="small" />
+        </Layout.FlexBox>
+
+        <Typography variant="h6">{title}</Typography>
+        <Layout.Spacer multiplier={0.5} />
+        <Typography variant="subtitle2" color={appColors.darkGrey}>
+          Last modified at 12:30 PM
+        </Typography>
```

Comment:
Move this to bottom somewhere, either shortened and in bottom right above the ID or as it is but maybe under the buttons?